app/controllers/admin/stats_controller.rb

```diff
@@ -1,5 +1,10 @@
 module Admin
   class StatsController < BaseController
+    # TO BE REMOVED if you use an active record scope in any actions of the
+    # controller.
+    before_action :skip_policy_scope
+    before_action -> { authorize(:stats) }
```

Comment:
Dans le application controller on a ça. 

```ruby
after_action :verify_policy_scoped, only: :index, unless: :skip_pundit?
```
Le scope est `enforced` uniquement sur l'index. Je pense que l'on peut simplifier avec un skip uniquement sur l'index et du coup pas de commentaire


```suggestion
    before_action :skip_policy_scope, only: [:index]
    before_action -> { authorize(:stats) }
```